const add = (num1, num2) => {
  return num1 + num2;
};

print(add(5 + 10));
console.log("I caused a merge conflict");
